### ISSUE TITLE
Wrong field descriptions

### ISSUE THREAD

**Issue by sungam3r** [graphql-dotnet/graphql-dotnet#1001]
## Description

Field descriptions are not displayed in the schema.

### Steps to reproduce

Create any schema with field descriptions and print it:
```csharp
 new SchemaPrinter(schema, new SchemaPrinterOptions
            {
                IncludeDescriptions = true,
                IncludeDeprecationReasons = true,
            }).Print();
```

### Expected result

Schema like
```
type PersonType {
  # BlaBlaBla1
  name: String!
  # BlaBlaBla2
  age: Int
  # BlaBlaBla3
  address: String
}
```

### Actual result

```
type PersonType {
  name: String!
  age: Int
  address: String
}
```

### Environment

Windows 10, .NET Core 2.1 console app

### Cause of error

https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Utilities/SchemaPrinter.cs#L254
and
https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Utilities/SchemaPrinter.cs#L255

Instead of `type.Description` and `type.DeprecationReason` there should be `x.Description` and `x.DeprecationReason`.

Also if you specify a description on the type, the `SchemaPrinter` prints all the fields with the same description from type ignoring descriptions from fields.

**Comment by joemcbride** [comment]
This is fixed by #983

**Comment by SimonCropp** [comment]
#983 is merged. so closing this one
